recipes/nasm/all/conanfile.py

```diff
@@ -64,12 +63,11 @@ def generate(self):
 
     def build(self):
         apply_conandata_patches(self)
-
+        autotools = Autotools(self)
```

Comment:
as I said it makes little sense, just wait NMakeToolchain of conan 1.55.0

Reply:
Hi @SpaceIm,
Okay, I've updated the recipe to use NMakeToolchain. Hopefully CCI will upgrade to 1.55.0 soon